spinalcordtoolbox/centerline/core.py

```diff
@@ -175,9 +176,30 @@ def get_centerline(im_seg, param=ParamCenterline(), verbose=1, remove_temp_files
         im_centerline.data = np.zeros(im_centerline.data.shape)
         # Assign value=1 to centerline. Make sure to clip to avoid array overflow.
         # TODO: check this round and clip-- suspicious
```

Comment:
We may want to move / rewrite this comment, since it only applies to to the `binary` case?

Reply:
Right! Comment moved in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4026/commits/30cb6c83801ab8098ef04e1243cdbd2197a535de. I also removed the TODO. After playing with this part of the code, I don't think that "round and clip" is suspicious -- it is working as expected.